logging/telemetryConfig.go

```diff
@@ -120,6 +130,22 @@ func (cfg TelemetryConfig) getInstanceName() string {
 	return fmt.Sprintf("%s:", pathHashStr[:16])
 }
 
+func SanitizeTelemetryString(input string, maxParts int) string {
+	parts := strings.SplitN(input, ":", maxParts)
+	for i := range parts {
+		// Limit to alphanumeric, and _-
+		if reg != nil {
+			parts[i] = reg.ReplaceAllString(parts[i], "")
+		}
+
+		// Limit length to support up to an ipv6 address
+		if len(parts[i]) > 39 {
```

Comment:
instead of using the magic number 39 ( which is well described the line above ), use the following
```const ipv6AddressLength := 39```
instead.